molecule/tomcat/prepare.yml

```diff
@@ -1,12 +1,21 @@
 ---
 - name: Prepare
   hosts: all
-  become_user: root
+  vars_files:
+    - vars.yml
   tasks:
     - name: Display Ansible version
       debug:
         msg: "Ansible version is  {{ ansible_version.full }}"
 
     - name: Install required dependencies
       package:
-        name: sudo
\ No newline at end of file
+        name: sudo
+
+  post_tasks:
+    - name: "Download latest Apache Tomcat zip file from {{ download_url }}."
+      get_url:
```

Comment:
@RanabirChakraborty tiny detail, but can you add the missing ansible.builtin. in front of each tasks (here, in front of get_url, but also elsewhere in the file)

Reply:
Thanks! done , pleas review once.